agent/envoy_bootstrap/platforminfo/platform_info_collector.go

```diff
@@ -350,7 +350,7 @@ func getEc2InstanceMetadata(query string) (string, error) {
 	}
 }
 
-func runCommand(name string, args ...string) (string, error) {
+func RunCommand(name string, args ...string) (string, error) {
```

Comment:
Curious why we need to capital R here

Reply:
I guess in Golang func name starting with small case makes it private and capital case makes it public. Now with this change we are using the RunCommand outside this file.

Reply:
This function belongs to a better place IMO. Can we have a dedicated util class for it? Along with the `EnvoyCLI` interface.

Reply:
I agree we can move it to util folder along with EnvoyCLI & FileUtil interface.

Is it ok to do these refactor as follow up, so that this change remains simple?

Reply:
Totally, we can have a separate PR for this.